gymnasium/envs/box2d/car_racing.py

```diff
@@ -121,10 +121,17 @@ class CarRacing(gym.Env, EzPickle):
     and turn at the same time.
 
     ## Action Space
-    If continuous:
-        There are 3 actions: steering (-1 is full left, +1 is full right), gas, and breaking.
-    If discrete:
-        There are 5 actions: do nothing, steer left, steer right, gas, brake.
+    If continuous there are 3 actions (S, G, B):
+    - (S) steering, -1 is full left, +1 is full right
```

Comment:
Could you add the numbers next to the actions

Reply:
done